OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -307,7 +307,7 @@ bool IsBlockedBy(Actor self, Actor otherActor, Actor ignoreActor, CellConditions
 			// If we have a real actor, we can then perform the extra checks that allow us to avoid being blocked.
 			if (self == null)
 				return true;
-
+						
```

Comment:
This line is triggering the travis style error.

Reply:
No, problem there, will get rid of it.